client/CFocusableHelper.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+ * CFocusableHelper.cpp, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#include "CFocusableHelper.h"
```

Comment:
Btw #include "StdInc.h" here might be needed. Sometimes I observe strange compilation errors without it. So if checks passed maybe it is ok

Reply:
Nothing needs to be included from the stdinc.